tests/unit/webapi.test.ts

```diff
@@ -21,18 +22,31 @@ const rs: RequestSpec = {
   service_in_group: { value: 'APT_Updates', negated: true },
 };
 
+// filter query that is sent to checkmk for graph and autocompleters
 const context: Context = {
   host: { host: 'ut_hostname' },
   siteopt: { site: 'ut_site' },
   hostregex: { host_regex: 'ut_host_regex' },
   service: { service: 'Check_MK' },
   serviceregex: { service_regex: 'ut_service_regex' },
-  host_labels: { host_label: '[{"value":"cmk/site:cmk210d"}]' },
+  host_labels: {
+    host_labels_1_bool: 'and',
+    host_labels_1_vs_1_bool: 'and',
+    host_labels_1_vs_1_vs: 'cmk/site:cmk210d',
+    host_labels_1_vs_count: '2',
```

Comment:
`host_labels_1_vs_count: '1',` right?

Reply:
nice! i used the output generated by the ui as a basis (which creates an additional element at the end), but seems i mixed something up.

Reply:
Yeah, the UI needs to always render an additional empty element. But in configs like this we don't need to add that empty field